common/src/main/resources/assets/wynntils/lang/en_us.json

```
@@ -1932,6 +1935,21 @@
   "screens.wynntils.poiManagementGui.setMarkers": "Set Markers",
   "screens.wynntils.poiManagementGui.setMarkers.tooltip": "Sets a marker at all selected POIs",
   "screens.wynntils.poiManagementGui.undo": "Undo delete",
+  "screens.wynntils.savedItems.addTooltip1": "Click to add a new category to your vault.",
+  "screens.wynntils.savedItems.addTooltip2": "Enter the name of the category you want to add then hit enter or click this button again to save it.",
+  "screens.wynntils.savedItems.changeCategoryTooltip1": "Left click to move the selected item to the current category.",
+  "screens.wynntils.savedItems.changeCategoryTooltip2": "Right click to cancel.",
+  "screens.wynntils.savedItems.deleteTooltip1": "Left click to delete current category and move all items in the category to no category",
+  "screens.wynntils.savedItems.deleteTooltip2": "Shift-Left click to delete current category and delete all items in current category",
```

```suggestion
  "screens.wynntils.savedItems.deleteTooltip1": "Left click to delete current category and move all items in the category to no category.",
  "screens.wynntils.savedItems.deleteTooltip2": "Shift-Left click to delete current category and delete all items in current category.",
```

More consistent this way